src/models/root.ts

```diff
@@ -0,0 +1,29 @@
+// Import: libs
+import * as React from "react";
+import { types, Instance, castToSnapshot } from "mobx-state-tree";
+
+// Import: components and models
+import { TransactionStore } from "./transaction";
+import { MessageStore } from "./message";
+import { ConfigurationStore } from "./configuration";
+
+// Stores init
+const transactionStore = TransactionStore.create();
+const messageStore = MessageStore.create();
+const configurationStore = ConfigurationStore.create();
+
+// RootStore init
+const RootStore = types.model("RootStore", {
+  transactionStore: types.optional(TransactionStore, {}),
+  messageStore: types.optional(MessageStore, {}),
+  configurationStore: types.optional(ConfigurationStore, {}),
+});
+export interface IRootStore extends Instance<typeof RootStore> {}
+
+const rootStore = RootStore.create({
```

Comment:
Tu n'as pas besoin de ce faire ces cast, sauf erreur, tu peux juste mettre des objets vides

Reply:
Understood, will be updated during future model refactor.